### ISSUE TITLE
To have something like jhbuild build and jhbuild buildone

### ISSUE THREAD

**Issue by nacho** [hexchat/gtk-win32#18]
Basically right now what happens is that you either build all the packages or you build one package with all the dependencies.

I'd actually would like to have something like "build.ps1 build gtk" which would basically be the same as "build.ps1 -OnlyOne gtk" but also have another command "build.ps1 buildone gtk" which would take care of only building gtk without the dependencies.

I did not see anything like that already implemented but let me know if I am wrong or if this feature is a WONTFIX.

**Comment by bviktor** [comment]
Obviously, you _can't_ build gtk without the dependencies.

**Comment by nacho** [comment]
You can if they are already built.

**Comment by Arnavion** [comment]
Yes, TingPing asked for this one a long time ago but I never got around to doing it.

**Comment by nacho** [comment]
@Arnavion should I try to give this a try?
